alexis.py

```diff
@@ -356,6 +365,20 @@ def init(self):
             resp = 'Hay {} {}:\n```\n{}\n```'.format(num_memes, word, '\n'.join(memelist))
             await self.send_message(chan, resp)
 
+        #cleverbot (@bot <mensaje>)
+        elif text.startswith('<@{}>'.format(self.user.id)):
```

Comment:
Agregaste lineas demás, como bien dice @jkcgs solo tenias que agregar `and self.cbotcheck != None` al `if`.